ECMA2Yaml/ECMAHelper/Models/ReflectionItem.cs

```diff
@@ -83,7 +83,7 @@ public string CommentId
         }
         public List<TypeParameter> TypeParameters { get; set; }
         public List<Parameter> Parameters { get; set; }
-        public Parameter ReturnValueType { get; set; }
+        public VersionedProperty<Parameter> ReturnValueType { get; set; }
```

Comment:
As mentioned in the docs-ui PR, all different versions of the return value will share the same description, so it's a bit too heavy if we monikerize the entire "Parameter" model. Also, now that we know parameter types won't change and return value type can change, it's starting to make more sense for us to give return value its only memory model, what do you think?